linode/lke/schema_resource.go

```diff
@@ -79,6 +79,11 @@ var resourceSchema = map[string]*schema.Schema{
 					Description: "A Linode Type for all of the nodes in the Node Pool.",
 					Required:    true,
 				},
+				"disk_encryption": {
+					Type:        schema.TypeString,
+					Description: "The disk encryption policy for the nodes in this pool.",
+					Computed:    true,
```

Comment:
Disk encryption will always be enabled for LKE clusters created in LDE-enabled regions, so this field is not user-configurable.